### PR TITLE
style(UI): set default fonts for LTR, RTL

### PR DESCRIPTION
**Pull request by adeelshahid**
- remove index.css. App.css contains the related code
- set default fonts for LTR and RTL

### CODE REVIEW

File: src/core/UI/base/typography.css [hunk 1]
```diff
@@ -11,6 +11,16 @@
...
  */
 
+
+
+[dir="ltr"] body, [dir="ltr"] div {
+    font-family: Roboto, sans-serif;
+}
+
+[dir="rtl"] body, [dir="rtl"] div {
+    font-family: "Noto Naskh Arabic", sans-serif;
```
Comment (varl) [inline src/core/UI/base/typography.css]: have you added a package for this to `package.json` as well?

Comment (adeelshahid) [reply src/core/UI/base/typography.css]: I wanted your perspective on this as we talked briefly about i18n last week.

Currently we use `import 'typeface-roboto'`, but there is no `typeface-noto-naskh-arabic` or similar available.

If we can control these fonts as small packages published on our _namespaced_ `@dhis2/typeface-${font-face}` repo we can add consistency at this layer.

Comment (varl) [reply src/core/UI/base/typography.css]: if we add a typeface package to the `@dhis2` scope we could move the `typeface-roboto` dep to that package as well and bundle all of our typefaces in a single package.

Comment (adeelshahid) [reply src/core/UI/base/typography.css]: sounds like a good idea

**Pull request merged**